R/DTW.R

```diff
@@ -14,7 +14,8 @@
 #'   evaluation grid for the second QTS.
 #' @param distance_only A boolean specifyung whether to only compute distance
 #'   (no backtrack, faster). Default is \code{FALSE}.
-#'
+#' @param step_pattern The choice for the step pattern of the warping path (see
+#'  \code{\link[dtw]{stepPattern}} for more information)
```

Comment:
Please end documentation with a proper period at the end of the sentence and specify what is the choice by default with a specific sentence like "Defaults to `dtw::symmetric2`" and explain briefly what this default does.